_data/changelogs/component-pagination.yml

```diff
@@ -2,6 +2,14 @@ title: Pagination
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Updated text-decoration style on pagination links so page numbers are underlined.
+  summaryAdditional: Link styles are now visually consistent with usa-link.
```

Comment:
**suggestion (non-blocking):** Added a suggestion to help improve clarity a bit. Let me know if you feel like this confuses anything. 

```suggestion
  summary: Added text underline styles to pagination links.
  summaryAdditional: Pagination links are now visually consistent with other USWDS text links.
```

Reply:
Resolved in [5ceadcc](https://github.com/uswds/uswds-site/pull/2793/commits/5ceadcc8b3f66a39f9e4fefece9e3cd9863f559f)